lib/generateSvg.js

```diff
@@ -0,0 +1,187 @@
+'use strict'
+const path = require('path')
+const { writeFile } = require('fs')
```

Comment:
I'd suggest using `mz/fs`. It has `writeFile` that returns a promise. In that case you can use the async/await syntax

Reply:
I don't know if this is necessary as the SVG is not an inline SVG. It's loaded in Markdown by reference and therefore we don't have to `await` for the file to be written.

Reply:
In that case, I think you can just return the generated svg and do the savings in `index.js`, with a `await Promise.all(svgs.map....)`

By the way, I see there's an error of mine in `index.js`, I do `writeFile` w/o a callback